csrankings-v.csv

```diff
@@ -447,7 +447,7 @@ Volkan Cevher,EPFL,https://people.epfl.ch/volkan.cevher,hlWhzU8AAAAJ
 Volkan Isler,University of Minnesota,http://www-users.cs.umn.edu/~isler,Q5KT-hEAAAAJ
 Volker Blanz,University of Siegen,http://www.grk1564.uni-siegen.de/de/blanz-volker,jYCidWgAAAAJ
 Volker Brendel,Indiana University,http://brendelgroup.org,w9oekBcAAAAJ
-Volker Dellwo,University of Zurich,https://www.cl.uzh.ch/de/people/team/phonetics/vdellw.html,HQlIBz8AAAAJ
+Volker Dellwo,University of Zurich,https://www.cl.uzh.ch/de/about-us/people/team/phonetics/vdellw.html,HQlIBz8AAAAJ

```

Comment:
Another non-CS. Please clarify how they meet the inclusion criteria as directed by the instructions, or remove.